src/common/common-util/src/test/java/ee/ria/xroad/common/conf/globalconf/ConfigurationLocationTest.java

```diff
@@ -37,15 +39,27 @@
  * Unit tests for {@link ConfigurationLocation}
  */
 public class ConfigurationLocationTest {
+    private static final String TLS_CERTIFICATION_VERIFICATION_ENABLED = "xroad.configuration-client.global_conf_tls_cert_verification";
+    private static final String HOSTNAME_VERIFICATION_ENABLED = "xroad.configuration-client.global_conf_hostname_verification";
 
     /**
      * Checks that {@link ConfigurationLocation} uses connections that timeout after a period of time.
      * @throws IOException
      */
     @Test
     public void connectionsTimeout() throws IOException {
-        URLConnection connection = ConfigurationLocation.getDownloadURLConnection("http://test.download.com");
+        URLConnection connection = ConfigurationLocation.getDownloadURLConnection("https://test.download.com");
```

Comment:
Maybe some some other URL could be used for testing? There's actual connection opened there...